src/java.base/share/classes/sun/security/provider/SHA3.java

```diff
@@ -65,8 +65,12 @@ abstract class SHA3 extends DigestBase {
     };
 
     private final byte suffix;
-    private byte[] state = new byte[WIDTH];
-    private long[] lanes = new long[DM*DM];
+    private long[] state = new long[DM*DM];
+
+    // The following two arrays are allocated to size WIDTH bytes, but we only
```

Comment:
Only one of the following arrays is of WIDTH length... outdated comment?

Reply:
Well, DM*DM*8 is WIDTH (DM=5 and WIDTH=200), so both arrays have 200 bytes, But I will update the comment to make it clearer.